pylintrc

```diff
@@ -284,7 +284,7 @@ argument-naming-style=snake_case
 
 # Regular expression matching correct argument names. Overrides argument-
 # naming-style.
-#argument-rgx=
+argument-rgx=(?P<exempt>[a-z_]{1,2})
```

Comment:
Not sure about this. I believe there's a good reason for the rule rejecting short names in general. They're fine for very short methods (or for contexts and lambda expressions, in which cases they are already allowed), but in general the scope of a name can be long enough that a short name unnecessarily makes the code less clear. (Same is true for variable names, below).